packages/popover2/src/contextMenu2.tsx

```diff
@@ -174,6 +174,7 @@ export const ContextMenu2: React.FC<ContextMenu2Props> = React.forwardRef<any, C
         menu === undefined ? undefined : (
             <Popover2
                 {...popoverProps}
+                autoFocus={false}
```

Comment:
this is the crux of the bugfix, but it caused a regression in https://github.com/palantir/blueprint/issues/4742